src/app/api/users/[email]/inventories/[historyId]/route.ts

```diff
@@ -19,4 +24,69 @@ export async function GET(request: NextRequest) {
         console.error("Erreur MySQL :", error);
         return NextResponse.json({ error: infoMessages.server }, { status: 500 });
     }
+}
+
+export async function POST(request: NextRequest) {
+    const { pathname } = request.nextUrl;
+    const pathParts = pathname.split("/");
+    const historyId = pathParts.pop();
+    const email = pathParts.at(-2);
+
+    try {
+        const body = await request.json();
+        const { objectId } = body;
+
+        if (!objectId) {
+            return NextResponse.json({ error: "objectId manquant" }, { status: 400 });
+        }
+
+        await db.query(
+            'INSERT INTO inventory (is_used, user_id, object_id, history_id) VALUES (0, (SELECT id FROM users WHERE email = ?), ?, ?)',
+            [email, objectId, historyId]
+        );
+
+        return NextResponse.json({ message: "Objet ajouté à l'inventaire." });
+    } catch (error) {
+        console.error("Erreur MySQL :", error);
+        return NextResponse.json({ error: infoMessages.server }, { status: 500 });
+    }
+}
+
+export async function DELETE(request: NextRequest) {
+    const { pathname } = request.nextUrl;
+    const pathParts = pathname.split("/");
+    const historyId = pathParts.pop();
+    const email = pathParts.at(-2);
+
+    try {
+        const body = await request.json();
+        const { objectIds } = body;
+
+        if (!Array.isArray(objectIds) || objectIds.length === 0) {
+            return NextResponse.json({ error: "Liste d'objets vide ou invalide." }, { status: 400 });
+        }
+
+        const [rows] = await db.query<UserRow[]>(
+            "SELECT id as userId FROM users WHERE email = ?",
```

Comment:
you already have the userId, don't need to add this request. If you want validation and a secure route, rather check the cookie with user credentials to compare with the userId given in the route